src/core/lib/security/credentials/tls/grpc_tls_certificate_provider.cc

```diff
@@ -21,11 +21,13 @@
 #include <grpc/support/alloc.h>
 #include <grpc/support/log.h>
 #include <grpc/support/string_util.h>
+#include <openssl/base.h>
 #include <openssl/ssl.h>
 
 #include "src/core/lib/gprpp/stat.h"
 #include "src/core/lib/slice/slice_internal.h"
 #include "src/core/lib/surface/api_trace.h"
+#include "tls_utils.h"
```

Comment:
Judging by the lines immediately above this one, the project's style seems to be src/core/... include paths, not a relative one.

Reply:
Thanks, noted.